vulnfeeds/cmd/alpine/run_alpine_convert.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
```

Comment:
I'm not exactly sure where this is intended to be run (adding some comments wouldn't hurt to help clarify this).

It would be safer to use `/bin/sh` instead. Disregarding what https://google.github.io/styleguide/shellguide.html#which-shell-to-use says about Bash, it does say to use `set -e` rather than options on the shebang line...

Reply:
Moved the -e into a `set -e` line. Let's stick with bash for now since all the other scripts are running bash as well. 

This is intended to be the entrypoint of the docker image. I've added a comment at the top of the two script files clarifying this.